src/configs/shell.rs

```diff
@@ -27,6 +28,7 @@ impl<'a> Default for ShellConfig<'a> {
             ion_indicator: "ion",
             elvish_indicator: "esh",
             tcsh_indicator: "tsh",
+            unknown_indicator: "",
```

Comment:
Does it make sense to use the question mark here as default? :thinking:

Reply:
Leaving it empty would be, in my opinion, slightly less jarring than a question mark.

Reply:
I guess you are right, especially because this is a new option.